src/components/AppNavigation/EditCalendarModal.vue

```diff
@@ -220,6 +245,7 @@ export default {
 				if (this.calendarColorChanged) {
 					await this.saveColor()
 				}
+				await this.saveTransparency()
```

Comment:
nit: only save if it changed?

Reply:
We have that here: https://github.com/nextcloud/calendar/blob/e98cd45aa9ae45477de8a2ccefb4c7a3fb43ed71/src/store/calendars.js#L549

I thought it would make more sense for the check to happen in the store

Reply:
That works too!